plugin_tests/client/mockVGL.js

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable camelcase */
+/* eslint-disable underscore/prefer-constant */
```

Comment:
@matthewma7 with recent release of geojs, we no longer need to maintain a copy of this 

@manthey